### PR TITLE
update getting started - take 1

### PR DESCRIPTION
**Pull request by ritzvik**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -45,3 +45,10 @@
 EinsteinPy is an open source pure Python package dedicated to problems arising in General Relativity and relativistic physics, such as goedesics plotting for schwartzschild space-time model, calculation of schwartzschild radius for any mass given. Features like visualisation of geodesics of curved black holes and 3D visualisations are some of the features which are planned.
 It is released under the MIT license.
 
+Contents
+--------
+
+.. toctree::
```
Comment (shreyasbapat) [inline docs/source/index.rst]: Its great that you figured out the `toctree`

Comment (ritzvik) [reply docs/source/index.rst]: made bhavya co-author of the commit. Merge after the tests get passed

Comment (ritzvik) [reply docs/source/index.rst]: @shreyasbapat travis build failed due to their connection error.

**Comment by shreyasbapat** [comment]
Will merge after the outage is over.
https://www.traviscistatus.com/

**Pull request merged**